src/oc/lib/api/common.clj

```diff
@@ -249,6 +286,10 @@
 (defn open-company-anonymous-resource [passphrase]
   (merge open-company-resource (anonymous-resource passphrase)))
 
+(defn open-company-id-token-resource [passphrase]
+  (merge (open-company-anonymous-resource passphrase)
+         (id-token-resource passphrase)))
+
```

Comment:
Also this should be like `open-company-authenticated-resource` instead of starting from the anonymous resource and adding the id-token-resource.